src/reconcile/rec_visibility.c

```diff
@@ -138,7 +142,7 @@ __rec_append_orig_value(
  * __rec_need_save_upd --
  *     Return if we need to save the update chain
  */
-static bool
+static inline bool
```

Comment:
I don't feel strongly, but why did you decide to inline a bunch of functions here? Did you see a performance problem?

Reply:
I don't feel anything. I just feel these functions are small and can be inlined.